youtube_callback/sftp_id_retrieve.py

```diff
@@ -48,21 +50,22 @@ def callfunction(course):
         shutil.rmtree(workdir)
     os.mkdir(workdir)
 
-    xml_downloader(course)
+    xml_csv_downloader(course)
 
     for file in os.listdir(workdir):
-        upload_data = domxml_parser(file)
+        if 'report-' in file:
+            upload_data = domxml_parser(file) if is_xml_file(file) else csv_parser(file)
 
```

Comment:
@ssemenova Are we consistently getting `report-` now or do we need to retrieve both `report-` and `status-`?

Reply:
Consistently getting report. I haven't seen status show up so far, I think they've phased it out.